contributors/devel/sig-architecture/feature-gates.md

```diff
@@ -152,17 +179,70 @@ other action outside of Kubernetes to use it. This gives some grace period for
 users to take action, but such feature gates will eventually set
 `LockToDefault` to `true` and then be retired, like normal.
 
-[After at least two releases post-GA and deprecation](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecation),
-feature gates should be removed. Typically, we add a comment in
+[After at least three releases where a feature has been locked to the default value, feature gates should be removedpost-GA/deprecation and locked to
+default](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecation),
+feature gates should be removed. We use three releases because it corresponds to
+roughly one year in the Kubernetes development cycle which is our
+[support period](https://kubernetes.io/releases/patch-releases/#support-period).
+
+To achieve our [compatibility version](#compatibility-versions) promise, all
```

Comment:
The first sentence here is just a recap of the previous?

Reply:
it refers to references rather than the gate itself. Is it repeating too much of the same information?